### PR TITLE
Feature/avs 421 82 calculate tax for virtual orders

### PR DESCRIPTION
**Pull request by sjb9774**

### CODE REVIEW

**Comment by rsisco** [comment]
@sjb9774 - Be sure to test this in a Magento 2.1 environment. Also, does this work correctly with guest checkout?

**Comment by sjb9774** [comment]
@rsisco I've updated the code and tested locally on 2.1.15 and 2.2.6 and it seems to work both

File: Observer/CalculateVirtualOrder.php [hunk 1]
```diff
@@ -84,22 +84,18 @@ public function execute(\Magento\Framework\Event\Observer $observer)
             try {
```
Comment (rsisco) [inline Observer/CalculateVirtualOrder.php]: @sjb9774 - FYI - I refactored this code slightly for conciseness.

**Comment by rsisco** [comment]
This PR addresses issue #121.

**Pull request merged**